interface/views.py

```diff
@@ -153,6 +154,7 @@ def done(request, pk):
         log.warning('Score is None')
 
     submission.score = points
```

Comment:
To be sure use `int(points)`, even though `score` is a `IntegerField`, python might still do something funny.